cloudify_gcp/compute/instance.py

```diff
@@ -80,14 +96,21 @@ def create(self):
         :raise: GCPError if there is any problem with startup script file:
         e.g. the file is not under the given path or it has wrong permissions
         """
-        self.logger.info(
-            'Create instance with parameters: {0}'.format(self.to_dict()))
+
+        if not utils.is_manager_instance():
+            add_to_security_groups(self)
+        disk = ctx.instance.runtime_properties.get(constants.DISK)
+        if disk:
+            self.disks = [disk]
+        if not self.disks and not self.image:
+            raise NonRecoverableError("A disk image ID must be provided")
 
         return self.discovery.instances().insert(
```

Comment:
I got RecoverableError but this is confusing for the end user. I agree it should be changed in check_response to handle 'is not ready'